pkg/core/interop/runtime/engine.go

```diff
@@ -74,7 +74,7 @@ func Notify(ic *interop.Context) error {
 		Name:       name,
 		Item:       stackitem.DeepCopy(stackitem.NewArray(args)).(*stackitem.Array),
 	}
-	ic.Notifications = append(ic.Notifications, ne)
+	ic.Notifications = append(ic.Notifications, ne) // Do not use ic.VM.AddRef, because `args` item was counted earlier.
```

Comment:
But we make a `DeepCopy` of it, so technically it's a different element. Also, `elem` is popped off the stack already.

Reply:
You're right, in C# it is done a bit earlier during `args` conversion to stackitem:
https://github.com/neo-project/neo/blob/ed68274e7dec2622468d3bc67df6e0f2913d903b/src/neo/SmartContract/ApplicationEngine.cs#L404

Reply:
Fixed.